lib/debops-tools/debops/config.py

```diff
@@ -39,7 +42,7 @@
 
 DEBOPS_CONFIG = ".debops.cfg"
 
-DEFAULTS = """
+DEFAULTS = u"""
```

Comment:
In Python strings support Unicode by default which I think is a good default. I guess you could leave it at:

```Python
DEFAULTS = """
```

? This is more common in Python 3 projects I would say.

Reply:
But it breaks the IOString stuff.
In py2 the default string type is a bytestring. Using u""" is more precise and should work on both versions.

Reply:
Works for me too then :) I remember, I also worked with IOString already.